web/satellite/src/views/Dashboard.vue

```diff
@@ -34,12 +32,14 @@
         PROJECT_USAGE_ACTIONS,
         BUCKET_USAGE_ACTIONS, PROJECT_PAYMENT_METHODS_ACTIONS
     } from '@/utils/constants/actionNames';
-    import ROUTES from '@/utils/constants/routerConstants';
-    import ProjectCreationSuccessPopup from '@/components/project/ProjectCreationSuccessPopup.vue';
-    import { AppState } from '../utils/constants/appStateEnum';
-    import { RequestResponse } from '../types/response';
-    import { User } from '../types/users';
+    import { AuthToken } from '@/utils/authToken';
+    import DashboardHeader from '@/components/header/Header.vue';
+    import NavigationArea from '@/components/navigation/NavigationArea.vue';
```

Comment:
disordered

Reply:
This is correct imports sort order. Alphabetical